src/opendr/control/multi_object_search/dependencies.ini

```diff
@@ -0,0 +1,10 @@
+[runtime]
+python=torch==1.9.0
+       torchvision==0.10.0
+       tensorboard
+       numpy
```

Comment:
@ad-daniel should max version be applied in every tool dependencies? If so we should add it here too.

Reply:
good point, yes we should in case the tool is installed separately

Reply:
@ad-daniel can you confirm whether this is ok? It seems that many comments were resolved but i can't see any changes.